### ISSUE TITLE
About retrain's accuracy

### ISSUE THREAD

**Issue by guanfaqian** [wujcan/GIF-torch#2]
I controlled the ratio of training nodes to test nodes to be roughly 1:9, in order to have roughly the same number of training sets as the default for cora, citeseer, etc. But in this couple, why is the accuracy of my code to execute Retrain only about 50%, theoretically it should be 70%, shouldn't it?
